src/LibraryManager.Build/Contracts/Logger.cs

```diff
@@ -21,16 +22,16 @@ public void Clear()
             Errors.Clear();
         }
 
-        public void Log(string message, Contracts.LogLevel level)
+        public void Log(string message, LogLevel level)
         {
             switch (level)
             {
-                case Contracts.LogLevel.Error:
+                case LibraryManager.Contracts.LogLevel.Error:
```

Comment:
Simplify these uses of `LogLevel` too?  Thanks!

Reply:
Sorry for missing that.  Done.